include/crc32c/crc32c.h

```diff
@@ -59,6 +59,13 @@ inline uint32_t Crc32c(const char* data, size_t count) {
   return Extend(0, reinterpret_cast<const uint8_t*>(data), count);
 }
 
+#ifdef __cpp_lib_byte
```

Comment:
I'll change this to `#if defined()` in a follow-up commit, to match the style used across the library.